deploy/examples/csi/rbd/storageclass-radosnamespace-test.yaml

```
@@ -0,0 +1,66 @@
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph # namespace:cluster
+spec:
+  failureDomain: osd
+  replicated:
+    size: 1
+    # Disallow setting pool with replica 1, this could lead to data loss without recovery.
+    # Make sure you're *ABSOLUTELY CERTAIN* that is what you want
+    requireSafeReplicaSize: false
```

since changing from a test file
```suggestion
    requireSafeReplicaSize: true
```